.github/ISSUE_TEMPLATE/allow-use-to-reference-entire-files.md

```diff
@@ -0,0 +1,5 @@
+name: Allow use to reference entire files
+about: new issue
+title: "[SVG] <TITLE HERE>"
+labels: SVG
```

Comment:
```suggestion
labels: SVG Use reference entire file
```
Since there might be other SVG-related explainers in the future, I'd like to scope the label to just this feature.